File: Selenium/Project/Activity2.java
package lmsproject;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;

public class Activity2 {
    public static void main(String[] args) {


        WebDriverManager.firefoxdriver().setup();
        WebDriver driver= new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/lms/");

        String text=driver.findElement(By.xpath("//h1[@class='uagb-ifb-title']")).getText();


        Assert.assertEquals("Learn from Industry Experts",text,"not matched with expected result");

        driver.close();


    }
}

File: Selenium/Project/Activity2.py
from _ast import Assert

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://alchemy.hguy.co/lms/")

    element= driver.find_element(By.XPATH,"//h1[@class='uagb-ifb-title']")
    assert "Learn from Industry Experts"==element.text
   